lib/Figure/FigurePreview.js

```diff
@@ -12,53 +12,54 @@ const FileCardPreview = ({
   const loadingText = progress === 100 ? 'Processing' : 'Uploading';
 
   if (!previewSrc) {
-    return <div className="file-card__title">{fileExtension}</div>;
+    return <div className="figure__title">{fileExtension}</div>;
   }
 
   const previewBeingProcessed = previewSrc && progress;
   if (previewBeingProcessed) {
     return (
-      <div className="file-card__title">
-        <div className="file-card__loader-wrapper">
-          <Icon name="loader" className="file-card__loader" />
-          <span className="file-card__progress">{progress}%</span>
+      <div className="figure__title">
+        <div className="figure__loader-wrapper">
+          <Icon name="loader" className="figure__loader" />
+          <span className="figure__progress">{progress}%</span>
         </div>
 
-        <div className="file-card__loader-text">{loadingText}</div>
+        <div className="figure__loader-text">{loadingText}</div>
       </div>
     );
   }
 
   const loadingPreview = previewSrc && !progress && !showPreview;
   if (loadingPreview) {
     return (
-      <div className="file-card__title">
-        <Icon name="loader" className="file-card__loader" />
+      <div className="figure__title">
+        <Icon name="loader" className="figure__loader" />
       </div>
     );
   }
-
+  // TODO: Add alt-text prop
```

Comment:
Is this being done before merging in yeah?

Reply:
Ah this was actually done, just forgot to remove the TODO

Reply:
Turns it it wasn't, it is now!